### DISCUSSION TITLE
Option to keep Tab panels mounted

### DISCUSSION THREAD

**Discussion by Angusross111** [mantinedev/mantine#1545] [Feature requests]
Currently each panel is only mounted when it is navigated to. There are situations where having all panels mounted is favourable. 
[Here](https://github.com/mui/material-ui/issues/21250) is a discussion in the MUI repo around different options, also some considerations around features React 18 could offer (pre-loading, offscreen rendering, etc).

**Comment by rtivital** [comment]
It will be added in 5.0
